src/rpc/net.cpp

```diff
@@ -322,6 +323,61 @@ static RPCHelpMan addnode()
     };
 }
 
+static RPCHelpMan addconnection()
+{
+    return RPCHelpMan{"addconnection",
+        "\nOpen an outbound connection to a specified node. This RPC is for testing only.\n",
+        {
+            {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "The IP address and port to attempt connecting to."},
+            {"connectiontype", RPCArg::Type::STR, RPCArg::Optional::NO, "'outbound' or 'blockrelay'."},
```

Comment:
0c4d67371 argument name in snakecase and improved description
```suggestion
            {"connection_type", RPCArg::Type::STR, RPCArg::Optional::NO, "The type of connection to open, either (case insensitive) \"outbound\" or \"blockrelay\"."},
```

Reply:
done